build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -39,19 +39,21 @@
       <PathSeparator>/</PathSeparator>
       <PathSeparator Condition=" '$(OSName)' == 'win' ">\</PathSeparator>
       <TestPackageBuildVersionSuffix>$(CommitCount)</TestPackageBuildVersionSuffix>
-      <TestOutputDir>$(BaseOutputDirectory)/tests</TestOutputDir>
-      <TestPackagesBuildDir>$(TestOutputDir)/packagesBuild</TestPackagesBuildDir>
-      <TestPackagesDir>$(TestOutputDir)/packages</TestPackagesDir>
-      <TestArtifactsDir>$(TestOutputDir)/artifacts</TestArtifactsDir>
-      <TestResultXmlDir>$(TestOutputDir)/results</TestResultXmlDir>
-      <TestDirectory>$(RepoRoot)/test</TestDirectory>
-      <DotnetUnderTest>$(DotnetStage2)</DotnetUnderTest>
+      <TestOutputDir>$(BaseOutputDirectory)/tests/</TestOutputDir>
+      <TestPackagesBuildDir>$(TestOutputDir)/packagesBuild/</TestPackagesBuildDir>
+      <TestPackagesDir>$(TestOutputDir)/packages/</TestPackagesDir>
+      <TestArtifactsDir>$(TestOutputDir)/artifacts/</TestArtifactsDir>
+      <TestResultXmlDir>$(TestOutputDir)/results/</TestResultXmlDir>
+      <TestDirectory>$(RepoRoot)/test/</TestDirectory>
```

Comment:
Note, adding trailing slashes since that is the convention for directories in MSBuild.